lib/task-data/schemas/types-installos.json

```diff
@@ -319,6 +324,14 @@
             "type": "string",
             "description": "Extra (persistent) kernel boot parameters",
             "minLength": 1
+        },
+        "Timezone": {
+            "description": "The Timezone based on $TZ (Asia/Tehran)",
```

Comment:
based on `Asia/Tehran` seems a little odd, suggest using UTC or local

Reply:
Its just example in description not a default value , if you do not set its utc by default .